tests/misc_env/install_prereq.py

```diff
@@ -343,9 +345,21 @@ def registry_login(ceph, distro_ver):
                 "passwd": config["registry_credentials"]["password"],
             }
         )
-
+    ceph.exec_command(sudo=True, cmd="mkdir ~/.docker")
+    ceph.exec_command(sudo=True, cmd="touch ~/.docker/config.json")
+    auths_file_sudo = ceph.remote_file(sudo=True,
+        file_name="/root/.docker/config.json", file_mode="w"
+    )
+    auths = {}
+    for r in registries:
+        b64_auth = base64.b64encode(f"{r['user']}:{r['passwd']}".encode("ascii"))
+        auths[r['registry']] = {"auth": b64_auth.decode("utf-8")}
+    auths_dict = {"auths":auths}
+    auths_file_sudo.write(json.dumps(auths_dict, indent=4))
+    auths_file_sudo.flush()
     for r in registries:
         ceph.exec_command(
             cmd=f"sudo {container} login -u {r['user']} -p {r['passwd']} {r['registry']}",
             check_ec=True,
         )
+
```

Comment:
When we use `sudo=True`, it is created for root user. This step is required however, we would also need it for `cephuser`. With this file in place, we don't need to perform `login`